poll/translations/en/LC_MESSAGES/text.po

```diff
@@ -0,0 +1,186 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"POT-Creation-Date: 2016-04-07 16:17+0500\n"
+"PO-Revision-Date: 2016-04-07 16:17+0500\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: en\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.8.7\n"
+"X-Poedit-Basepath: ../../..\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Poedit-SearchPath-0: poll.py\n"
+
+#: poll.py
+msgid "Whether or not to display results to the user."
+msgstr ""
+
+#: poll.py
+msgid "The maximum number of times a user may send a submission."
+msgstr ""
+
+#: poll.py
+msgid "Number of times the user has sent a submission."
+msgstr ""
+
+#: poll.py
+msgid "Text to display after the user votes."
+msgstr ""
+
+#: poll.py
+#, python-brace-format
+msgid "'{0}' is not present, or not a JSON array."
+msgstr ""
+
+#: poll.py
+#, python-brace-format
+msgid "{0} {1} not a javascript object!"
```

Comment:
(Important, but optional if you consider this out of scope): Without context, it is difficult for translators to make sense of something like this. I think all occurrences of "{0}" and "{1}" in strings should be replaced with meaningful variables like "{noun} {item} not a javascript object" and include a translator comment like `# translators: {noun} is either "Answer" or "Question"`